### ISSUE TITLE
Build Error NETSDK1130 when using .Net 5.0 and NuGet-Package 'Microsoft.Toolkit.Forms.UI.Controls.WebView'

### ISSUE THREAD

**Issue by Lohnegrim** [windows-toolkit/Microsoft.Toolkit.Win32#279]
Compiling with .Net 5.0 Preview 6 raises this Error when then NuGet-Package 'Microsoft.Toolkit.Forms.UI.Controls.WebView' (Tested with Version 6.1.1) is referenced:
error NETSDK1130: Referencing a Windows Metadata component directly when targeting .NETCoreApp,Version=v5.0 is not supported. Use the C#/WinRT projection tool (https://aka.ms/cswinrt) or a provided projection for this target.

**Comment by michael-hawker** [comment]
@azchohfi I'd assume this is expected?

@Lohnegrim please try using [WebView2](https://aka.ms/webview2) instead here, that's the supported path moving forward.

**Comment by azchohfi** [comment]
Yes, this is expected. This package will not work as-is on DotNet5, as the old projections for winmds were removed in favor of cswinrt.

**Comment by Lohnegrim** [comment]
@michael-hawker 
I tryed to create a new project with WebView2 as explained in this Link:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/winforms
but didn't get the Webview2 Control in the Toolbox.
Not sure whats wrong, because I also tryed to replace 'Microsoft.Toolkit.Forms.UI.Controls.WebView' with 'Microsoft.Web.WebView2', in the project I used before, but it also didn't work (didn't find WebView2).
NuGet shows the that WebView2 is installed, but everywhere else I don't see it.

Remark: 
The Version I installed is 0.9.579
I restarted VS multiple times, rebuild multiple times opened and closed the Designer multiple times but I don't see the WebView2.

**Comment by carlossanlop** [comment]
@azchohfi I think I'm seeing the same issue @Lohnegrim is seeing.

I'm trying to upgrade an old .NET Framework WinForms project to .NET 5.0. I followed the [instructions for WebView2]( https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/winforms), but after downloading the nuget package, I am unable to add the using and consume the WebView2 type.

What information would you need from me to help investigate the problem?

**Comment by azchohfi** [comment]
That is a good question. Have you tried it with the new version that we shipped ~5 days ago?
https://www.nuget.org/packages/Microsoft.Web.WebView2/0.9.622.11
Maybe it was fixed.

**Comment by Lohnegrim** [comment]
Hi @azchohfi 
I think this may be a different issue, but I just tryed this with a new project in the latest VS2019 Version, and it didn't work.
At least till I unloaded the new project and loaded it again.
So I only got NuGet to Work after unloading and reloading the project, not sure why.
But even so I only worked with The Preview Version, and not with the Version you linked
